<html xmlns:tp="http://schemas.datacontract.org/2004/07/Microsoft.VisualStudio.TestPlatform.Extensions.HtmlLogger.ObjectModel" xmlns:a="http://schemas.microsoft.com/2003/10/Serialization/Arrays" xmlns:msxsl="urn:schemas-microsoft-com:xslt">
  <body>
    <h1>Test run details</h1>
    <div class="summary">
      <div class="block"><span>Total tests</span><div class="total-tests">15</div><br></div>
      <div class="block"><span>Passed  : </span><span class="passedTests">2</span><br><span>Failed  : </span><span class="failedTests">13</span><br><span>Skipped : </span><span class="skippedTests">0</span><br></div>
      <div class="block"><span>Pass percentage</span><div class="pass-percentage">13 %</div><br></div>
      <div class="block"><span>Run duration</span><div class="test-run-time">8m 36s</div><br></div><br></div>
    <h2>Failed Results</h2><details open=""><summary>C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\FileSyncServer.Tests.dll</summary><div class="inner-row" Id="2005047293-failedResult">
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked​</span><div class="duration"><span>36s 182ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs:line 88
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked​</span><div class="duration"><span>36s 87ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs:line 62
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses​</span><div class="duration"><span>32s 806ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs:line 136
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses​</span><div class="duration"><span>32s 76ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs:line 89
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "unknownfile.txt")​</span><div class="duration"><span>32s 63ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'unknownfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent1.txt")​</span><div class="duration"><span>32s 33ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent2.txt")​</span><div class="duration"><span>32s 35ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test3.txt")​</span><div class="duration"><span>32s 37ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test3.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test2.txt")​</span><div class="duration"><span>32s 25ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test1.txt")​</span><div class="duration"><span>32s 24ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetLargeFile_ReturnsFileContent​</span><div class="duration"><span>32s 39ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for large file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetLargeFile_ReturnsFileContent() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 106
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics​</span><div class="duration"><span>34s 543ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs:line 124
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses​</span><div class="duration"><span>32s 63ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file '1\test1.txt' from client 'ba775ab7-3fdd-4672-9a24-c9b2b948efaf'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs:line 196
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
      </div></details><h2>All Results</h2><details><summary>C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\FileSyncServer.Tests.dll</summary><div class="inner-row" Id="2005047293">
        <div class="leaf-division">
          <div><span class="pass">✔</span><span> FileSyncServer.Tests.ClientCachingTests.Client_Cache_Enabled_Should_Serve_File_After_Permission_Revoked​</span><div class="duration"><span>8s 984ms</span><br></div>
          </div>
          <div class="error-info"></div>
        </div>
        <div class="leaf-division">
          <div><span class="pass">✔</span><span> FileSyncServer.Tests.ClientCachingTests.Client_Cache_Disabled_Should_Fail_After_Permission_Revoked​</span><div class="duration"><span>5s 846ms</span><br></div>
          </div>
          <div class="error-info"></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked​</span><div class="duration"><span>36s 182ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs:line 88
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked​</span><div class="duration"><span>36s 87ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs:line 62
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses​</span><div class="duration"><span>32s 806ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs:line 136
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses​</span><div class="duration"><span>32s 76ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs:line 89
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "unknownfile.txt")​</span><div class="duration"><span>32s 63ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'unknownfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent1.txt")​</span><div class="duration"><span>32s 33ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent2.txt")​</span><div class="duration"><span>32s 35ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 81
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test3.txt")​</span><div class="duration"><span>32s 37ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test3.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test2.txt")​</span><div class="duration"><span>32s 25ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test1.txt")​</span><div class="duration"><span>32s 24ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName) in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 61
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.FileFetchingTests.GetLargeFile_ReturnsFileContent​</span><div class="duration"><span>32s 39ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for large file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.FileFetchingTests.GetLargeFile_ReturnsFileContent() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs:line 106
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics​</span><div class="duration"><span>34s 543ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs:line 124
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
        <div class="leaf-division">
          <div><span class="fail">✘</span><span> Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses​</span><div class="duration"><span>32s 63ms</span><br></div>
          </div>
          <div class="error-info">
    Error: <span class="error-message"><pre>Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file '1\test1.txt' from client 'ba775ab7-3fdd-4672-9a24-c9b2b948efaf'., but found HttpStatusCode.InternalServerError {value: 500}.</pre></span><br>
    Stack trace: <span class="error-stack-trace"><pre>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)
   at Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses() in C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs:line 196
--- End of stack trace from previous location ---</pre></span><br></div>
        </div>
      </div></details><div>
      <h2>Informational messages</h2><span>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.8.2+699d445a1a (64-bit .NET 9.0.0-rc.1.24431.7)</span><br><span>[xUnit.net 00:00:00.46]   Discovering: FileSyncServer.Tests</span><br><span>[xUnit.net 00:00:00.48]   Discovered:  FileSyncServer.Tests</span><br><span>[xUnit.net 00:00:00.49]   Starting:    FileSyncServer.Tests</span><br><span>Server started...</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: a5d39ac9-10c5-43b1-aa77-8518bdcba4ce</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: a5d39ac9-10c5-43b1-aa77-8518bdcba4ce</span><br><span>Read and Delete permissions for OMEN\budcr are already granted.</span><br><span>Inheritance disabled and inherited rules removed.</span><br><span>Server Output: Server cache enabled = True</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: e1b591a2-720a-4fb5-afa4-2a0878ed928b) for file: testfile.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: testfile.txt, requestId: e1b591a2-720a-4fb5-afa4-2a0878ed928b</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 80906695-5bc7-4c20-bd66-24880642052c) for file: testfile.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 20 bytes for file: testfile.txt, requestId: 80906695-5bc7-4c20-bd66-24880642052c</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: testfile.txt, requestId: 80906695-5bc7-4c20-bd66-24880642052c</span><br><span>Inheritance is already disabled.</span><br><span>Second Response Status: 200</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>Read and Delete permissions for OMEN\budcr are already granted.</span><br><span>Inheritance disabled and inherited rules removed.</span><br><span>Server Output: Server cache enabled = False</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: b858cac0-4761-486f-90a1-d06098d328a6) for file: testfile.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: testfile.txt, requestId: b858cac0-4761-486f-90a1-d06098d328a6</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 46240b6d-866f-4912-a692-f071b54d47cb) for file: testfile.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 20 bytes for file: testfile.txt, requestId: 46240b6d-866f-4912-a692-f071b54d47cb</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: testfile.txt, requestId: 46240b6d-866f-4912-a692-f071b54d47cb</span><br><span>Inheritance is already disabled.</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: a3127713-c95c-4020-b0cb-7a1d3ba5db1c) for file: testfile.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: testfile.txt, requestId: a3127713-c95c-4020-b0cb-7a1d3ba5db1c</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 3417368a-1438-4862-95ea-7e03878e5a3f) for file: testfile.txt</span><br><span>Client Output: fail: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Access denied to file 'testfile.txt'.</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>Client Output: Status(StatusCode="Unavailable", Detail="Error reading next message. IOException: The request was aborted. IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.. SocketException: An existing connection was forcibly closed by the remote host.", DebugException="System.IO.IOException: The request was aborted.")</span><br><span>Server started...</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: c7a23ad7-8cd9-46df-a291-e921a8f8602d</span><br><span>Read and Delete permissions for OMEN\budcr are already granted.</span><br><span>Inheritance disabled and inherited rules removed.</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: c7a23ad7-8cd9-46df-a291-e921a8f8602d</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Server cache enabled = False</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>Read and Delete permissions for OMEN\budcr are already granted.</span><br><span>Inheritance disabled and inherited rules removed.</span><br><span>[xUnit.net 00:01:17.30]       Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:01:17.30]       Stack Trace:</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:01:17.30]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:01:17.30]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs(88,0): at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked()</span><br><span>[xUnit.net 00:01:17.30]         --- End of stack trace from previous location ---</span><br><span>Server Output: Server cache enabled = True</span><br><span>Granted Read and Delete permissions to OMEN\budcr.</span><br><span>Inheritance is already disabled.</span><br><span>[xUnit.net 00:01:53.38]       Expected response1.StatusCode to be HttpStatusCode.OK {value: 200}, but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:01:53.38]       Stack Trace:</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:01:53.38]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:01:53.38]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ServerCaching.cs(62,0): at FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked()</span><br><span>[xUnit.net 00:01:53.38]         --- End of stack trace from previous location ---</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Server started...</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: c27aa22d-49c5-4591-a868-2fe0b79b13ab</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: c27aa22d-49c5-4591-a868-2fe0b79b13ab</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>[xUnit.net 00:02:38.67]       Expected result.Status to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:02:38.67]       Stack Trace:</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:02:38.67]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:02:38.67]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs(136,0): at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses()</span><br><span>[xUnit.net 00:02:38.67]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:03:10.75]       Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:03:10.75]       Stack Trace:</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:03:10.75]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:03:10.75]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\ConcurrentRequestsTests .cs(89,0): at Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses()</span><br><span>[xUnit.net 00:03:10.75]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:03:42.81]       Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'unknownfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:03:42.81]       Stack Trace:</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:03:42.81]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:03:42.81]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(81,0): at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName)</span><br><span>[xUnit.net 00:03:42.81]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:04:14.85]       Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:04:14.85]       Stack Trace:</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:04:14.85]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:04:14.85]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(81,0): at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName)</span><br><span>[xUnit.net 00:04:14.85]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:04:46.88]       Expected response.StatusCode to be HttpStatusCode.NotFound {value: 404} because Expected status code 404 for non-existing file 'nonexistent2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:04:46.88]       Stack Trace:</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:04:46.88]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:04:46.88]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(81,0): at Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(String fileName)</span><br><span>[xUnit.net 00:04:46.88]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:05:18.92]       Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test3.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:05:18.92]       Stack Trace:</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:05:18.92]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:05:18.92]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(61,0): at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName)</span><br><span>[xUnit.net 00:05:18.92]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:05:50.94]       Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test2.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:05:50.95]       Stack Trace:</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:05:50.95]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:05:50.95]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(61,0): at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName)</span><br><span>[xUnit.net 00:05:50.95]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:06:22.97]       Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for existing file 'test1.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:06:22.97]       Stack Trace:</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:06:22.97]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:06:22.97]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(61,0): at Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(String fileName)</span><br><span>[xUnit.net 00:06:22.97]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:06:55.01]       Expected response.StatusCode to be HttpStatusCode.OK {value: 200} because Expected status code 200 for large file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:06:55.01]       Stack Trace:</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:06:55.01]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:06:55.01]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\FileFetchingTests.cs(106,0): at Server.FileFetchingTests.GetLargeFile_ReturnsFileContent()</span><br><span>[xUnit.net 00:06:55.01]         --- End of stack trace from previous location ---</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Server started...</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: 9c2bc761-b5bb-49cb-acf3-93043302a6e1</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: 9c2bc761-b5bb-49cb-acf3-93043302a6e1</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: ef056a4e-2063-4b46-b2aa-e1bafbf986c5</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: ef056a4e-2063-4b46-b2aa-e1bafbf986c5</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: b49297a4-2a4f-4dac-9503-5960fadfa97f</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: b49297a4-2a4f-4dac-9503-5960fadfa97f</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: dec46dc9-183a-450f-854d-f1200a8526d9</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: dec46dc9-183a-450f-854d-f1200a8526d9</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: 39fdf318-58ce-41cb-ae2a-353a0f9c9e8a</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: 39fdf318-58ce-41cb-ae2a-353a0f9c9e8a</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: f33d58c1-448c-421b-8525-d74f0eedecfb</span><br><span>Created large test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\performance_test1.txt</span><br><span>Created large test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\performance_test2.txt</span><br><span>Created large test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\performance_test3.txt</span><br><span>Created large test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\performance_test4.txt</span><br><span>Created large test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\performance_test5.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: f33d58c1-448c-421b-8525-d74f0eedecfb</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Server Output: ThreadPool Stats:</span><br><span>Server Output: Worker Threads in Use: 1/768 (Min: 0)</span><br><span>Server Output: Completion Port Threads in Use: 1/768 (Min: 0)</span><br><span>Disposed client with clientId: 9c2bc761-b5bb-49cb-acf3-93043302a6e1</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: ef056a4e-2063-4b46-b2aa-e1bafbf986c5</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: b49297a4-2a4f-4dac-9503-5960fadfa97f</span><br><span>Disposed client with clientId: dec46dc9-183a-450f-854d-f1200a8526d9</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: 39fdf318-58ce-41cb-ae2a-353a0f9c9e8a</span><br><span>[xUnit.net 00:07:51.93]       Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file 'largeemptyfile.txt'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:07:51.93]       Stack Trace:</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:07:51.93]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:07:51.93]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs(124,0): at Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics()</span><br><span>[xUnit.net 00:07:51.93]         --- End of stack trace from previous location ---</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: c618761d-44c3-452e-b1f8-ed8f71dea93b</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: c618761d-44c3-452e-b1f8-ed8f71dea93b</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: ba775ab7-3fdd-4672-9a24-c9b2b948efaf</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: ba775ab7-3fdd-4672-9a24-c9b2b948efaf</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: e6a19d7d-d76c-4704-a7a2-69b002ac5c72</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: e6a19d7d-d76c-4704-a7a2-69b002ac5c72</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: d1b0f8a6-b412-42c4-b232-dcf209242f01</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: d1b0f8a6-b412-42c4-b232-dcf209242f01</span><br><span>Client Output: Starting FileClient...</span><br><span>Client Output: Checking server health...</span><br><span>Client Output: Server is healthy.</span><br><span>Client Output: Temporary directory already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test1.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test2.txt</span><br><span>Client Output: Test file already exists: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\test3.txt</span><br><span>Client Output: File synchronization initiated.</span><br><span>Client started with clientId: 3a1e07a3-2b26-4289-a999-f0b8d6661541</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test1.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test2.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test3.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent Init response to server with clientGuid: 3a1e07a3-2b26-4289-a999-f0b8d6661541</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test4.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test5.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test6.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test7.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test8.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test9.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\0\test10.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test1.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test2.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test3.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test4.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test5.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test6.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test7.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test8.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test9.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\1\test10.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test1.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test2.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test3.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test4.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test5.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test6.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test7.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test8.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test9.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\2\test10.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test1.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test2.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test3.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test4.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test5.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test6.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test7.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test8.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test9.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\3\test10.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test1.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test2.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test3.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test4.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test5.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test6.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test7.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test8.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test9.txt</span><br><span>Created test file: C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\bin\Debug\net9.0\client_cache\4\test10.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: f35c4f78-3df4-4462-8a45-118f35aece5c) for file: 0/test9.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 1a9d966e-6523-48de-9dab-e165a03d053c) for file: 0/test3.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 0ff31d97-b654-4d95-b9e8-a43a9703624f) for file: 0/test10.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 6f9e83e6-65a2-47ae-af0c-5a2e6e170caa) for file: 0/test8.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: d0b8399e-4f43-49c2-af71-ee0de31d2f14) for file: 0/test6.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 06f7f382-4d0d-4ae1-979b-007852c335b5) for file: 0/test7.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 2e598fdb-d4a0-495e-a1b8-83f162183196) for file: 0/test4.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: c31d4b48-6cb3-4cf8-9abd-743bbf19cdeb) for file: 0/test1.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: e8796332-7ca6-448d-b2a7-f3ab550ce27d) for file: 0/test5.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 7140b733-16a7-4af4-bed0-34da6e75c911) for file: 0/test2.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test8.txt, requestId: 6f9e83e6-65a2-47ae-af0c-5a2e6e170caa</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test3.txt, requestId: 1a9d966e-6523-48de-9dab-e165a03d053c</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test6.txt, requestId: d0b8399e-4f43-49c2-af71-ee0de31d2f14</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test1.txt, requestId: c31d4b48-6cb3-4cf8-9abd-743bbf19cdeb</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test7.txt, requestId: 06f7f382-4d0d-4ae1-979b-007852c335b5</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test10.txt, requestId: 0ff31d97-b654-4d95-b9e8-a43a9703624f</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test4.txt, requestId: 2e598fdb-d4a0-495e-a1b8-83f162183196</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test5.txt, requestId: e8796332-7ca6-448d-b2a7-f3ab550ce27d</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test2.txt, requestId: 7140b733-16a7-4af4-bed0-34da6e75c911</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 0/test9.txt, requestId: f35c4f78-3df4-4462-8a45-118f35aece5c</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 1863dd27-ac49-48bd-b5a5-db406a66d6ba) for file: 0/test8.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 6d58a3c7-1ac4-4d93-8c5b-7c12d4a7ef0e) for file: 0/test9.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 27bc26b4-c2b2-4726-b6b5-30e23f518d64) for file: 0/test3.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: b27d0ab2-f111-4d11-8019-890407e7c641) for file: 0/test6.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 0448013f-ece1-4a42-b857-90922cc91455) for file: 0/test1.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 920d10fd-0ebb-4cd4-a3ed-7a6084bf9fe6) for file: 0/test7.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 51559be8-7e3c-42dd-8805-8ffaeec6145f) for file: 0/test2.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: a473c354-9d8b-4df9-95d1-d2cdb6f48710) for file: 0/test10.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 342d0c8c-f982-47c1-a219-761a87e9a5f1) for file: 0/test5.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: b44c2af7-708c-4ee3-b9a5-3df7fd2239b2) for file: 0/test4.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test9.txt, requestId: 6d58a3c7-1ac4-4d93-8c5b-7c12d4a7ef0e</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test2.txt, requestId: 51559be8-7e3c-42dd-8805-8ffaeec6145f</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test5.txt, requestId: 342d0c8c-f982-47c1-a219-761a87e9a5f1</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test8.txt, requestId: 1863dd27-ac49-48bd-b5a5-db406a66d6ba</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 24c351f7-9fd7-43df-8bea-45746626aa3e) for file: 2/test1.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test3.txt, requestId: 27bc26b4-c2b2-4726-b6b5-30e23f518d64</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 277c2b7f-aaa9-498a-b19a-0ba2b6f3c33e) for file: 2/test7.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: eb6155ac-a7c6-4397-b98a-00f5cdb23f96) for file: 2/test5.txt</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test4.txt, requestId: b44c2af7-708c-4ee3-b9a5-3df7fd2239b2</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: ca561a30-13ff-4303-a4da-517b16b5a712) for file: 2/test6.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 8c1a7f41-1d7c-40d4-8a5c-9aaa447d4e74) for file: 2/test3.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: bf4e4a82-6dcb-4c79-a391-f1f56371cc8c) for file: 2/test4.txt</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test6.txt, requestId: b27d0ab2-f111-4d11-8019-890407e7c641</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test7.txt, requestId: 920d10fd-0ebb-4cd4-a3ed-7a6084bf9fe6</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 0/test1.txt, requestId: 0448013f-ece1-4a42-b857-90922cc91455</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: cf66cde0-c6e4-4da9-8f98-21a443097bb7) for file: 2/test10.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: 1ced8534-cfc0-4c1b-8766-5fa2cc6f27a7) for file: 2/test2.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test9.txt, requestId: 6d58a3c7-1ac4-4d93-8c5b-7c12d4a7ef0e</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test2.txt, requestId: 51559be8-7e3c-42dd-8805-8ffaeec6145f</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test3.txt, requestId: 27bc26b4-c2b2-4726-b6b5-30e23f518d64</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 51 bytes for file: 0/test10.txt, requestId: a473c354-9d8b-4df9-95d1-d2cdb6f48710</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test8.txt, requestId: 1863dd27-ac49-48bd-b5a5-db406a66d6ba</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test4.txt, requestId: b44c2af7-708c-4ee3-b9a5-3df7fd2239b2</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test6.txt, requestId: b27d0ab2-f111-4d11-8019-890407e7c641</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test5.txt, requestId: 342d0c8c-f982-47c1-a219-761a87e9a5f1</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test7.txt, requestId: 920d10fd-0ebb-4cd4-a3ed-7a6084bf9fe6</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: dd3d64d1-2608-4ec5-ab28-4b0c2a005d67) for file: 2/test9.txt</span><br><span>Client Output:       Completed file data transfer for file: 0/test1.txt, requestId: 0448013f-ece1-4a42-b857-90922cc91455</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 0/test10.txt, requestId: a473c354-9d8b-4df9-95d1-d2cdb6f48710</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received MetaData request (requestId: e1d44b52-0baa-485d-b85e-f9425121ee69) for file: 2/test8.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test9.txt, requestId: dd3d64d1-2608-4ec5-ab28-4b0c2a005d67</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test10.txt, requestId: cf66cde0-c6e4-4da9-8f98-21a443097bb7</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test6.txt, requestId: ca561a30-13ff-4303-a4da-517b16b5a712</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test3.txt, requestId: 8c1a7f41-1d7c-40d4-8a5c-9aaa447d4e74</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test1.txt, requestId: 24c351f7-9fd7-43df-8bea-45746626aa3e</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test7.txt, requestId: 277c2b7f-aaa9-498a-b19a-0ba2b6f3c33e</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test4.txt, requestId: bf4e4a82-6dcb-4c79-a391-f1f56371cc8c</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test5.txt, requestId: eb6155ac-a7c6-4397-b98a-00f5cdb23f96</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test2.txt, requestId: 1ced8534-cfc0-4c1b-8766-5fa2cc6f27a7</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent metadata for file: 2/test8.txt, requestId: e1d44b52-0baa-485d-b85e-f9425121ee69</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 81678be7-84e8-4286-9c74-685d70762f26) for file: 2/test10.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 027d2c8f-aa99-4509-87d1-5e33b027bdc8) for file: 2/test8.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: fb036b4e-4185-4b52-9555-874d4557b0ce) for file: 2/test9.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 3eba4d73-5333-4bb5-a810-88442114ee84) for file: 2/test6.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: e7af7844-7a94-4851-ba6b-0520f7e9cae5) for file: 2/test3.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 160abfff-3627-4d31-a405-7c48827ab5d1) for file: 2/test1.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: a79e6ff3-19f3-462c-81b7-e3108c443998) for file: 2/test4.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 56b91365-c07a-46eb-a471-75ddfa3a6a9a) for file: 2/test5.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: d0b4fae1-29a2-4657-9265-b0b2c07bb7b3) for file: 2/test2.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Received FileData request (requestId: 4a234819-b9c9-42a0-83f7-d05afdd4923f) for file: 2/test7.txt</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test9.txt, requestId: fb036b4e-4185-4b52-9555-874d4557b0ce</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test8.txt, requestId: 027d2c8f-aa99-4509-87d1-5e33b027bdc8</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test6.txt, requestId: 3eba4d73-5333-4bb5-a810-88442114ee84</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test1.txt, requestId: 160abfff-3627-4d31-a405-7c48827ab5d1</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test4.txt, requestId: a79e6ff3-19f3-462c-81b7-e3108c443998</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test2.txt, requestId: d0b4fae1-29a2-4657-9265-b0b2c07bb7b3</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test5.txt, requestId: 56b91365-c07a-46eb-a471-75ddfa3a6a9a</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 51 bytes for file: 2/test10.txt, requestId: 81678be7-84e8-4286-9c74-685d70762f26</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test3.txt, requestId: e7af7844-7a94-4851-ba6b-0520f7e9cae5</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Sent file chunk of size 50 bytes for file: 2/test7.txt, requestId: 4a234819-b9c9-42a0-83f7-d05afdd4923f</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test2.txt, requestId: d0b4fae1-29a2-4657-9265-b0b2c07bb7b3</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test4.txt, requestId: a79e6ff3-19f3-462c-81b7-e3108c443998</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test3.txt, requestId: e7af7844-7a94-4851-ba6b-0520f7e9cae5</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test1.txt, requestId: 160abfff-3627-4d31-a405-7c48827ab5d1</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test8.txt, requestId: 027d2c8f-aa99-4509-87d1-5e33b027bdc8</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test5.txt, requestId: 56b91365-c07a-46eb-a471-75ddfa3a6a9a</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test10.txt, requestId: 81678be7-84e8-4286-9c74-685d70762f26</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test9.txt, requestId: fb036b4e-4185-4b52-9555-874d4557b0ce</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test6.txt, requestId: 3eba4d73-5333-4bb5-a810-88442114ee84</span><br><span>Client Output: info: FileSyncClient.Services.ClientFileSyncManager[0]</span><br><span>Client Output:       Completed file data transfer for file: 2/test7.txt, requestId: 4a234819-b9c9-42a0-83f7-d05afdd4923f</span><br><span>Disposed client with clientId: c618761d-44c3-452e-b1f8-ed8f71dea93b</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: ba775ab7-3fdd-4672-9a24-c9b2b948efaf</span><br><span>Disposed client with clientId: e6a19d7d-d76c-4704-a7a2-69b002ac5c72</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: d1b0f8a6-b412-42c4-b232-dcf209242f01</span><br><span>Client Output: Status(StatusCode="Internal", Detail="Internal server error.")</span><br><span>Disposed client with clientId: 3a1e07a3-2b26-4289-a999-f0b8d6661541</span><br><span>[xUnit.net 00:08:36.36]       Expected result.Status to be HttpStatusCode.OK {value: 200} because Expected status code 200 for file '1\test1.txt' from client 'ba775ab7-3fdd-4672-9a24-c9b2b948efaf'., but found HttpStatusCode.InternalServerError {value: 500}.</span><br><span>[xUnit.net 00:08:36.36]       Stack Trace:</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)</span><br><span>[xUnit.net 00:08:36.36]            at FluentAssertions.Primitives.EnumAssertions`2.Be(TEnum expected, String because, Object[] becauseArgs)</span><br><span>[xUnit.net 00:08:36.36]         C:\Users\budcr\source\repos\RemoteBlazorWebView\src\Benchmarks\FilePOC\FileSyncServer.Tests\PerformanceTests .cs(196,0): at Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses()</span><br><span>[xUnit.net 00:08:36.36]         --- End of stack trace from previous location ---</span><br><span>[xUnit.net 00:08:36.36]   Finished:    FileSyncServer.Tests</span><br></div>
    <div>
      <h2>Error and Warning messages</h2><span>[xUnit.net 00:01:17.30]     FileSyncServer.Tests.ServerCachingTests.Server_Cache_Disabled_Should_Fail_After_Permission_Revoked [FAIL]</span><br><span>[xUnit.net 00:01:53.38]     FileSyncServer.Tests.ServerCachingTests.Server_Cache_Enabled_Should_Serve_File_After_Permission_Revoked [FAIL]</span><br><span>[xUnit.net 00:02:38.67]     Server.ConcurrentRequestsTests.MultipleConcurrentFileRequestsWithSameFile_ReturnsCorrectResponses [FAIL]</span><br><span>[xUnit.net 00:03:10.75]     Server.ConcurrentRequestsTests.MultipleConcurrentFileRequests_ReturnsCorrectResponses [FAIL]</span><br><span>[xUnit.net 00:03:42.81]     Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "unknownfile.txt") [FAIL]</span><br><span>[xUnit.net 00:04:14.85]     Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent1.txt") [FAIL]</span><br><span>[xUnit.net 00:04:46.88]     Server.FileFetchingTests.GetNonExistingFile_ReturnsNotFound(fileName: "nonexistent2.txt") [FAIL]</span><br><span>[xUnit.net 00:05:18.92]     Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test3.txt") [FAIL]</span><br><span>[xUnit.net 00:05:50.94]     Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test2.txt") [FAIL]</span><br><span>[xUnit.net 00:06:22.97]     Server.FileFetchingTests.GetExistingFile_ReturnsFileContent(fileName: "test1.txt") [FAIL]</span><br><span>[xUnit.net 00:06:55.01]     Server.FileFetchingTests.GetLargeFile_ReturnsFileContent [FAIL]</span><br><span>[xUnit.net 00:07:51.93]     Server.PerformanceTests.FetchLargeFiles_PerformanceMetrics [FAIL]</span><br><span>[xUnit.net 00:08:36.35]     Server.PerformanceTests.MultipleClientsConcurrentFileRequests_ReturnsCorrectResponses [FAIL]</span><br></div>
  </body>
  <style>
        body { font-family: Calibri, Verdana, Arial, sans-serif; background-color: White; color: Black; }
        h2 {
        margin-top: 15px;
        margin-bottom:10px;
        }
        pre {
        white-space: pre-wrap;
        }
        .summary {font-family:monospace;
        display: -webkit-flex; /* Safari */
        -webkit-flex-wrap: wrap; /* Safari 6.1+ */
        display: flex;
        flex-wrap: wrap;
        }
        .row {
        border: 2px solid #ffffff;
        width:100%;
        cursor:pointer;
        background-color:#d7e9fa;
        }
        .list-row {
        border: 2px solid #ffffff;
        width:100%;
        cursor:pointer;
        background-color:#ffffff;
        }
        .inner-row{
        border: 2px solid #ffffff;
        padding-left:1%;
        margin-left:1%;
        background-color:#ffffff;
        }
        .block{
        width : 150px;
        }
        .leaf-division{
        border: 2px solid #ffffff;
        background-color:#e6eff7;
        }
        .pass { color: #0c0; }
        .fail { color: #c00; }
        .error-message { color : brown; }
        .error-stack-trace { color: brown; }
        .duration {float:right;padding-right:1%;}
        .total-tests { font-size : 30px}
        .test-run-time { font-size : 30px}
        .pass-percentage { font-size : 30px}
        .error-info{
        margin-left:16px;
        }
      </style>
</html>